include/lib/output.inc.php

```diff
@@ -821,9 +821,9 @@ function getTranslatedCodeStr($codes) {
 
 			/* get $_msgs_new from the DB */
 			$sql	= 'SELECT * FROM '.TABLE_PREFIX.'language_text WHERE variable="_msgs" AND (language_code="'.$_SESSION['lang'].'" OR language_code="'.$parent.'")';
```

Comment:
Consider to move SQL executions into proper DAO classes when they appear outside of DAO classes, for example, `LanguageTextDAO.class.php` in this case.